tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/replication/RepeatStartBackupTest.java

```diff
@@ -156,8 +171,110 @@ public void testLoopStart() throws Exception {
 
          Assertions.assertEquals(0, errors.get());
       }
+   }
+
+   @Test
+   public void testAckManagerRepetition() throws Exception {
+
+      String queueName = "queue_" + RandomUtil.randomString();
+
+      server.getConfiguration().setMirrorAckManagerQueueAttempts(300000);
+      server.getConfiguration().setMirrorAckManagerRetryDelay(1000);
+      backupServer.getConfiguration().setMirrorAckManagerPageAttempts(300000);
+      backupServer.getConfiguration().setMirrorAckManagerRetryDelay(1000);
+
+      ExecutorService executorService = Executors.newFixedThreadPool(2);
+      runAfter(executorService::shutdownNow);
+
+      AtomicInteger errors = new AtomicInteger(0);
+      AtomicBoolean running = new AtomicBoolean(true);
+
+      runAfter(() -> running.set(false));
+      CountDownLatch latch = new CountDownLatch(1);
+      CountDownLatch backupStarted = new CountDownLatch(1);
+
+      AtomicInteger messagesSent = new AtomicInteger(0);
+
+      int starBackupAt = 100;
+      Assertions.assertFalse(server.isReplicaSync());
+      Assertions.assertFalse(backupServer.isStarted());

```

Comment:
We can lose the _Assertions._ prefixes (here and elsewhere) with imports, like the existing ones for other assert methods.

Reply:
I inteded to do that.. and I was going to do it...


I was working remotely yesterday (away from my desktop) and I forgot to include it.. my IDE was not working correctly this time.


But thanks for noticing.